backend/de.metas.swat/de.metas.swat.base/src/test/java/de/metas/invoicecandidate/api/impl/InvoiceCandBLCreateInvoicesTest.java

```diff
@@ -179,7 +181,10 @@ public void init()
 	public void test_InvalidInvoiceCandidate_NoUserInCharge_FlagItAsError()
 	{
 		final I_C_BPartner bpartner = icTestSupport.bpartner("test-bp");
+		final I_C_PaymentTerm paymentTerm =icTestSupport.paymentTerm("test-paymentTerm");
```

Comment:
I_C_Invoice_Candidate.C_PaymentTerm_ID is nullable.... 
* so pls avoid overcomplicating the tests if not needed
* if tests are failing might be because they are right, so not set payment term is not handled even though is allowed